src/main/java/nextstep/subway/auth/ui/AuthenticationPrincipalArgumentResolver.java

```diff
@@ -21,6 +22,9 @@ public boolean supportsParameter(MethodParameter parameter) {
     @Override
     public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
         Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        if(authentication==null){
```

Comment:
IDE의 포맷팅 기능으로 띄어쓰기등의 포맷팅을 해보면 좋을 것 같아요 😃 
```suggestion
        if(authentication == null){
```